Src/DependencyCollector/Net45.Tests/DependencyTrackingTelemetryModuleTest.cs

```diff
@@ -64,7 +79,20 @@ public void TestHttpPostRequestsAreCollected()
         public void TestHttpRequestsWithQueryStringAreCollected()
         {
             ITelemetry sentTelemetry = null;
-            var channel = new StubTelemetryChannel { OnSend = telemetry => sentTelemetry = telemetry };
+
+            var channel = new StubTelemetryChannel
```

Comment:
I tried to factor this out, but the lambda capture of sentTelemetry makes it hard to put it in a separate method.
Non-ideal, but leaving it this way may be the easiest.